doc/data/messages/c/condition-evals-to-constant/good.py

```diff
@@ -1 +1,2 @@
-# This is a placeholder for correct code for this message.
+CONSTANT = 100
+CONSTANT or True
```

Comment:
This message probably means that there was a mistake so the "good" example can be different I think:
```suggestion
def is_a_fruit(fruit):
    return fruit in {"apple", "orange"}
```